scripts/build_darwin.sh

```diff
@@ -10,9 +10,7 @@ mkdir -p dist
 # If installed to an alternate location use the following to enable
 # export SDKROOT=/Applications/Xcode_12.5.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
 # export DEVELOPER_DIR=/Applications/Xcode_12.5.1.app/Contents/Developer
-export CGO_CFLAGS=-mmacosx-version-min=11.3
-export CGO_CXXFLAGS=-mmacosx-version-min=11.3
-export CGO_LDFLAGS=-mmacosx-version-min=11.3
+export CGO_CPPFLAGS=-mmacosx-version-min=11.3
 
 rm -rf llama/build dist/darwin-*
 
```

Comment:
A few lines below this...

I'm not seeing how `make -j 8 dist` is going to work anymore, so this script seems like it needs more work.

I would also have expected to see changes in `./scripts/build_windows.ps1`